miniapps/shifted/marking.cpp

```diff
@@ -103,6 +103,50 @@ void ShiftedFaceMarker::MarkElements(const ParGridFunction &ls_func,
    initial_marking_done = true;
 }
 
+void ShiftedFaceMarker::MarkFaces(mfem::Array<int> &elem_marker,
+                                  mfem::Array<int> &face_marker)
+{
+   pmesh.ExchangeFaceNbrNodes();
+
+   face_marker.SetSize(pmesh.GetNumFaces());
+   face_marker=SBFaceType::UNDEFINED;
+   // Set face_attribute = attr to faces that are on the border between INSIDE and CUT elements.
+   for (int f = 0; f < pmesh.GetNumFaces(); f++)
+   {
+      auto *ft = pmesh.GetFaceElementTransformations(f, 3);
```

Comment:
Can we rewrite lines 116-127 as lines 137-144.

Reply:
This was a code left from other branch. Now it is cleaned.